OracleTSAM/samples/xe-db-svc/README.md

```diff
@@ -0,0 +1,42 @@
+Running Oracle TSAM Plus with oracle/database:11.2.0.2-xe image
+=============================
+You can run the TSAM container with the `oracle/database:11.2.0.2-xe` image provided on [GitHub](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+
+Due to the `oracle/database:11.2.0.2-xe` container creates new database and generates random `sys` user password on startup, a little "hacking" is needed here to make the `sys` password fixed.
```

Comment:
This should be rephrased. There is no hacking needed to provide a common password for the database container. The automatic generated password was put in place as per Oracle Security Policy. It can be set to whatever is desired by using the `setPassword.sh` script inside the container.